ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/action/Separator.java

```diff
@@ -22,6 +21,7 @@
     private String myText;
 
     public Separator() {
+        this(null);
```

Comment:
I thought It is null by default :) But Oleksii told me that there is some problem with undefined and null value. Thanks for explanation.